impl/src/metadata.rs

```diff
@@ -98,8 +100,9 @@ pub struct RazeMetadata {
   // `cargo metadata` output of the current project
   pub metadata: Metadata,
 
-  // The absolute path to the current project's cargo workspace root
-  pub workspace_root: PathBuf,
+  // The absolute path to the current project's cargo workspace root. Note that the workspace
+  // root in `metadata` will be inside of a temporary directory.
+  pub cargo_workspace_root: PathBuf,
```

Comment:
Can you add a reference to 

https://doc.rust-lang.org/cargo/reference/workspaces.html#root-package

Here? My brain was failing to parse "cargo_workspace_root" because my brain was stuck to thinking of workspace root as "bazel root".